mkat_fpga_tests/test_cbf.py

```diff
@@ -517,10 +563,10 @@ def get_actual_phases():
                 # set coarse delay on correlator input m000_y
                 # correlator_fixture.katcp_rct.req.delays time.time+somethign
                 # See page 22 on ICD ?delays on CBF-CAM ICD
-                reply, informs = correlator_fixture.katcp_rct.req.input_labels()
-                source_name = reply.arguments[1:][0].split()
+                reply,informs = correlator_fixture.katcp_rct.req.input_labels()
+                source_name = reply.arguments[1].split()
```

Comment:
informs: returns an empty list.
reply: returns katcp message with status and message are merged into one
hence, the code below.

Resolution:
Jira ticket was made.